custom/motioncontrollers/BangBangController.java

```diff
@@ -18,6 +19,7 @@
 	protected double A;
 	protected double F;
 	protected double threshold;
+	protected static final String DEFAULT_SMARTDASHBOARD_PREFIX = "!!";
```

Comment:
This did not exist before. It is for the bang (!) bang (!) controller. We should make a shebang controller at some point.